src/libstd/collections/hash/table.rs

```diff
@@ -126,9 +126,10 @@ unsafe impl<K: Send, V: Send> Send for RawTable<K, V> {}
 unsafe impl<K: Sync, V: Sync> Sync for RawTable<K, V> {}
 
 pub struct RawBucket<K, V> {
-    hash: *mut HashUint,
+    _hash: *mut HashUint,
```

Comment:
Why was this renamed? If it was to disambiguate with the method, I think `hash_` would be more appropriate than `_hash`, since the standard practice is to use prefix `_` to denote "unused."